frontend/src/components/icons/IconBase.vue

```diff
@@ -51,6 +51,9 @@ export default {
   computed: {
     iconColorCode () {
       const iconColor = this.iconColor
+      if (/^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$/.test(iconColor)) {
```

Comment:
```suggestion
      if (/^#([a-f0-9]{6}|[a-f0-9]{3})$/i.test(iconColor)) {
```

Reply:
case insensitive regex

Reply:
then we should do it in all places or even better move the regex to utils

Reply:
moving it to utils makes sense